code/modules/jobs/job_types/captain.dm

```diff
@@ -59,6 +59,29 @@
 
 	id_trim = /datum/id_trim/job/captain
 
+/datum/outfit/job/captain/pre_equip(mob/living/carbon/human/H, visualsOnly)
+	. = ..()
+	SSmapping.HACK_LoadMapConfig()
+	var/list/job_changes = SSmapping.config.job_changes
+	if(!length(job_changes))
+		return
+	var/list/captain_changes = job_changes["captain"]
+	if(!length(captain_changes))
+		return
+	var/special_charter = captain_changes["special_charter"]
+	if(!special_charter)
+		return
+	backpack_contents = list(/obj/item/melee/classic_baton/telescopic = 1) //removes normal charter
+	switch(special_charter)
+		if(CAPTAIN_MOON_CHARTER)
+			l_hand = /obj/item/station_charter/banner/moon
+		if(CAPTAIN_ASTEROID_CHARTER)
+			l_hand = /obj/item/station_charter/banner/asteroid
```

Comment:
I'd rather not add a completely separate typepath if it's just going to be one value changed. Just change it on post_equip, or whatever it is.